backend/fms_core/services/sample.py

```diff
@@ -243,14 +245,18 @@ def transfer_sample(process: Process,
 
             derived_samples_destination = sample_source.derived_samples.all()
             volume_ratios = {}
+            projects = {}
             for derived_sample in derived_samples_destination:
-                volume_ratios[derived_sample.id] = DerivedBySample.objects.get(sample=sample_source, derived_sample=derived_sample).volume_ratio
+                source_derived_by_sample = DerivedBySample.objects.get(sample=sample_source, derived_sample=derived_sample)
+                volume_ratios[derived_sample.id] = source_derived_by_sample.volume_ratio
+                projects[derived_sample.id] = source_derived_by_sample.project
```

Comment:
you're getting `source_derived_by_sample` from `derived_samples_destination`?

Reply:
Never mind, the sample is source, the derived sample is what the source and destination sample have in common.